azerothcore/azerothcore-wotlk
# fix(Core/Misc): Remove from LFG/batleground queues when teleported to…

… battleground/LFG dungeon.

Players cannot queue up for battleground during LFG dungeon.
Players cannot queue up for LFG dungeon during battleground/arena match.
Fixes #10635



## Issues Addressed:

- Closes #10635

## Tests Performed:

- Not tested.

## How to Test the Changes:

JoinBGAndLFG.Enable = 1 in worldserver conf
.debug lfg
queue up bg
Queue up lfg
accept invite lfg
see that bg queue is gone


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
